roles/cobbler/tasks/main.yml

```diff
@@ -1,4 +1,78 @@
 ---
+- name: Include cobbler keys.
+  include_vars: "{{ secrets_path | mandatory }}/cobbler_keys.yml"
+  no_log: true
+  tags:
+    - vars
+
+- name: Create /root/.ssh
+  file:
+    path: /root/.ssh
+    mode: '700'
+    state: directory
+
+- name: Write cobbler keys
+  copy:
+    content: "{{ item.data }}"
+    dest: "{{ item.path }}"
+    mode: '600'
+  with_items: cobbler_keys["{{ ansible_hostname}}"]
+  no_log: true
+
+- name: Include package type specific vars.
+  include_vars: "{{ ansible_pkg_mgr }}_systems.yml"
+  tags:
+    - always
+
+- include: yum_systems.yml
+  when: ansible_pkg_mgr == "yum"
+
+- include: apt_systems.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Start cobbler
+  service:
+    name: "{{ cobbler_service }}"
+    state: started
+    enabled: yes
+
+- name: Enable tftpd
+  lineinfile:
```

Comment:
If it's not an issue then that's awesome!  I was just remembering I ran into an issue with the testnodes role where if I was using 'regexp' and 'line' that 'line' would be added on every run if 'regexp' was found, even if 'line' already existed.  Adding 'backrefs' fixed that.